recipes-support/simple-http-server/files/simple-http-server.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Simple built-in http server in python
+Wants=network.target
+
+[Service]
+Type=oneshot
+ExecStart=/usr/sbin/simple-http-server.sh
```

Comment:
I think you need to run popd in ExecStartPost section. So that the directory you pushed will be poped out.